back/node_watcher/src/tasks/createBlockNumber.ts

```diff
@@ -37,7 +37,7 @@ const createBlockNumber: Task<NomidotBlock> = {
 
     const write = await prisma.createBlockNumber({
       number: blockNumber.toNumber(),
-      authoredBy: authoredBy.toString(),
+      authoredBy: authoredBy.toHex(),
```

Comment:
Still to be discussed, but maybe saving public key is better?

Reply:
Actually, I'm still hesitant. Addresses will be way easier to inspect

Reply:
maybe we just be redundant and have two columns in there - (public_key, address)?